pkg/apis/autoscaling/v1beta1/machineautoscaler_types.go

```diff
@@ -10,15 +10,21 @@ func init() {
 
 // MachineAutoscalerSpec defines the desired state of MachineAutoscaler
 type MachineAutoscalerSpec struct {
+	// MinReplicas constrains the minimal number of replicas
```

Comment:
s/minimal number of replicas to be present all the time in the cluster during scale down?

Reply:
Maybe `MinReplicas constrains the minimal number of replicas of a scalable resource`?

Reply:
done